salt_stack/salt/python/pyminifier.sls

```diff
@@ -6,7 +6,7 @@ include:
 pyminifier:
     pip.installed:
         - name: pyminifier==2.1
-        - bin_env: /usr/local/bin/pip3
+        - pip_bin: /usr/bin/pip3
         - require:
             - sls: python.python35

```

Comment:
These v3.5 requirements aren't causing problems?

Reply:
Ah I missed that require, looks like the pyminifier salt file is only used in the "workstation" packer AMI though so it might not be executed when creating the core AMIs for the stack...Not sure what "workstation" is for? A developer environment I'm guessing?

```
    'workstation*':
        - python.python35
        - git
        - vault.client
        - aws.boto3
        - python.pyminifier
 ```